modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -181,7 +181,7 @@ export class {{classname}} extends runtime.BaseAPI {
             {{#hasBodyParam}}
             {{#bodyParam}}
             {{#isContainer}}
-            body: requestParameters.{{paramName}}.map({{#items}}{{datatype}}{{/items}}ToJSON),
+            body: JSON.stringify(requestParameters.{{paramName}}{{#isListContainer}}{{#items}}{{^isPrimitiveType}}.map({{datatype}}ToJSON){{/isPrimitiveType}}{{/items}}{{/isListContainer}}),
```

Comment:
The body is a string now. Does this work correctly?

Reply:
According to the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch):


> `body`: Any body that you want to add to your request: this can be a Blob, BufferSource, FormData, URLSearchParams, or USVString object. Note that a request using the GET or HEAD method cannot have a body.

Passing random objects and lists on my local results in a nonsense body value of `[object Object]`.  I believe this has always been broken.

Reply:
strange, but how can this
https://github.com/OpenAPITools/openapi-generator/blob/fae073874b3f1a5350f4e31c4168c7d97b8921be/samples/client/petstore/typescript-fetch/builds/default/apis/PetApi.ts#L95
work, it is also a object

Reply:
it is converted here:
https://github.com/OpenAPITools/openapi-generator/blob/fae073874b3f1a5350f4e31c4168c7d97b8921be/samples/client/petstore/typescript-fetch/builds/default/runtime.ts#L63-L65
so JSON.stringify is not necessary here

Reply:
I see - did not catch that, thanks!